exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/MetricAdapter.java

```diff
@@ -228,40 +239,22 @@ private static Temporality mapToTemporality(Descriptor descriptor) {
 
   // TODO: Consider to pass the Builder and directly add values.
   @SuppressWarnings("MixedMutabilityReturnType")
-  static List<ValueAtPercentile> toProtoValueAtPercentiles(
-      Collection<MetricData.ValueAtPercentile> valueAtPercentiles) {
-    if (valueAtPercentiles.isEmpty()) {
-      return Collections.emptyList();
-    }
-    List<ValueAtPercentile> result = new ArrayList<>(valueAtPercentiles.size());
+  static void addBucketValues(
+      List<MetricData.ValueAtPercentile> valueAtPercentiles,
+      DoubleHistogramDataPoint.Builder builder) {
+
     for (MetricData.ValueAtPercentile valueAtPercentile : valueAtPercentiles) {
-      result.add(
-          ValueAtPercentile.newBuilder()
-              .setPercentile(valueAtPercentile.getPercentile())
-              .setValue(valueAtPercentile.getValue())
-              .build());
+      // TODO(jkwatson): Value of histogram should be long?
```

Comment:
@jkwatson :)

Reply:
I have no remembrance of what this might have meant.